src/goSuggest.ts

```diff
@@ -44,6 +44,14 @@ export class GoCompletionItemProvider implements vscode.CompletionItemProvider {
 		return this.provideCompletionItemsInternal(document, position, token, vscode.workspace.getConfiguration('go'));
 	}
 
+	private isInBlockComment(document: vscode.TextDocument, position: vscode.Position): boolean {
+		let text = document.getText();
+		let currentOffset = document.offsetAt(position);
+		let lastBlockCommentStartIndex = text.lastIndexOf('/*', currentOffset);
+		let lastBlockCommentEndIndex = text.lastIndexOf('*/', currentOffset);
+		return lastBlockCommentStartIndex > lastBlockCommentEndIndex;
```

Comment:
@thomasdarimont So I finally had time to look at your changes to `go-outline`.

Am not very comfortable changing the structure of the output, we will have to maintain 2 different ways of consuming the output, one for people still on the old version of `go-outline`, the other for the new.

Why not re-use the `Declaration` struct for comments. You can have `type` as comments, `label` as the comment text, `Children` would be empty array.

This way the final output structure would be the same as before.
An outer declaration for `package` whose children would declarations who in turn can be functions, types, variables, constants and comments

We can have a flag `-include-comments`. So that existing users of this tool will see no change in the usage even if they got the latest. We would return the comments, if this flag has been provided.